openpgp/packet/userattribute.go

```diff
@@ -59,7 +83,13 @@ func NewUserAttributePhoto(photos ...image.Image) (uat *UserAttribute, err error
 func NewUserAttribute(contents ...*OpaqueSubpacket) *UserAttribute {
 	return &UserAttribute{Contents: contents}
 }
-
+func (uat *UserAttribute) Data() []byte {
+	buf := bytes.NewBuffer(nil)
+	for _, osp := range uat.Contents {
+		osp.Serialize(buf)
+	}
+	return buf.Bytes()
+}
```

Comment:
This seems like a weird/dangerous API to me. Shouldn't the caller process the images one by one?

If it's only meant to be used in `userAttributeSignatureHash`, maybe let's make it private/unexported?